shell/platform/windows/key_event_handler.cc

```diff
@@ -13,15 +13,36 @@
 static constexpr char kChannelName[] = "flutter/keyevent";
 
 static constexpr char kKeyCodeKey[] = "keyCode";
+static constexpr char kScanCodeKey[] = "scanCode";
+static constexpr char kCharacterCodePointKey[] = "characterCodePoint";
+static constexpr char kModifiersKey[] = "modifiers";
 static constexpr char kKeyMapKey[] = "keymap";
 static constexpr char kTypeKey[] = "type";
 
-static constexpr char kAndroidKeyMap[] = "android";
+static constexpr char kWindowsKeyMap[] = "windows";
 static constexpr char kKeyUp[] = "keyup";
 static constexpr char kKeyDown[] = "keydown";
 
 namespace flutter {
 
+// Re-definition of the modifiers for compatibility with the Flutter framework.
+// These have to be in sync with the framework's RawKeyEventDataWindows
+// modifiers definition.
+const int kShift = 1 << 0;
+const int kShiftLeft = 1 << 1;
+const int kShiftRight = 1 << 2;
+const int kControl = 1 << 3;
+const int kControlLeft = 1 << 4;
+const int kControlRight = 1 << 5;
+const int kAlt = 1 << 6;
+const int kAltLeft = 1 << 7;
+const int kAltRight = 1 << 8;
+const int kWinLeft = 1 << 9;
+const int kWinRight = 1 << 10;
+const int kCapsLock = 1 << 11;
+const int kNumLock = 1 << 12;
+const int kScrollLock = 1 << 13;
```

Comment:
Put all this in an anonymous namespace for internal linkage.

Maybe consider an enum class for grouping, but IIRC that may mean you'll need to do casting when you `|` them all together, in which case it's probably not worth it.

Reply:
Yeah I would still have to cast. I just moved them to the namespace.